bin/run.js

```diff
@@ -3,5 +3,5 @@
 // eslint-disable-next-line unicorn/prefer-top-level-await
 (async () => {
 	const oclif = await import('@oclif/core')
-	await oclif.execute({development: true, dir: __dirname})
+	await oclif.execute({dir: __dirname})
```

Comment:
This script runs with node instead of ts-node, which was causing some weird illegitimate compile errors. Removing the `development` property means it no longer auto-compiles before every execution, to sidestep those errors. `bin/dev.js` still auto-compiles, which is why we should switch to using that when we're running locally.